docs/database/usage/index.md

```diff
@@ -335,3 +335,38 @@ and caching. To disable this functionality, update the `database_persistence_ena
 ```
 
 To enable persistence, view the [Offline Support](/database/offline-support) documentation.
+
+# Change the database URL
+
+## Switch the database URL of the default app
+
+If the default installed firebase instance needs to address a different database within the same project, call the database method on the default app with passing the database URL. 
+For example: 
+
+```js
+import firebase from '@react-native-firebase/app';
+import '@react-native-firebase/database';
```

Comment:
Lets go with the single import like the other examples:

```
import database '@react-native-firebase/database';
```

Reply:
Calling `firebase.app().database('http://your-url-string.com')` requires the general import of `import '@react-native-firebase/database';`
=> otherwise a warning is thrown: "You attempted to use "firebase.app('[DEFAULT]').database" but this module could not be found"

At least this happened to me. Is there another way to do it?

See the reason for this suggested implementation [in the comment below](https://github.com/invertase/react-native-firebase/pull/3431#discussion_r406701830)

Reply:
Can do:

```js
import { firebase } from '@react-native-firebase/database';
```

Reply:
Thanks. Done.